### PR TITLE
feat: add some plugins

### LINKED ISSUE #1379: [Website]: Plugins page is incomplete
### Current State

**Description of the current state/issue**:
APISIX already supports 80+ plugins, but the PluginHub page only shows more than 40 plugins. 
**Link to the page**: https://apisix.apache.org/plugins/

**Screenshots (if relevant)**:

### Desired State

**Description of the change you are proposing**:

We could add the introduction information of other plugins to this page.

### Would you like to fix this issue?

_No response_

### PR DESCRIPTION
**Pull request by uniqie**
Fixes: #1379 
Changes:
   Add some plugins

Remainning problems: 
  Some plugins  added missing pictures.

Screenshots of the change:
image

### CODE REVIEW

**Comment by SkyeYoung** [comment]
@uniqie could u add default plugin img? Also could u improve the components? I know you have great technical skills. 😇

File: website/static/data/plugins.json [hunk 1]
```diff
@@ -86,131 +184,180 @@
...
-        "name": "ext-plugin-post-req",
-        "description": "The ext-plugin-post-req runs after executing builtin Lua plugins and before proxying to the upstream",
-        "useDefaultIcon": true
+        "name": "kafka-proxy",
+        "description": "The kafka-proxy plugin can be used to configure advanced parameters for the kafka upstream of Apache APISIX, such as SASL authentication"
       }
     ]
   }
-]
+]
```
Comment (starsz) [inline website/static/data/plugins.json]: Better to add EOL.

**Comment by uniqie** [comment]
> @uniqie could u add default plugin img? Also could u improve the components? I know you have great technical skills. 😇
You r too highly of me. 🐶

File: website/static/data/plugins.json (hunk 1 above)
Comment (uniqie) [reply website/static/data/plugins.json]: I can't get you. Where is need EOL?

Comment (SkyeYoung) [inline website/static/data/plugins.json]:
```suggestion
]

```

Comment (SkyeYoung) [reply website/static/data/plugins.json]: just here

**Comment by SkyeYoung** [comment]
@uniqie Okay, could u help us create an issue to track this?

**Review by SkyeYoung — APPROVED** [review]

**Comment by uniqie** [comment]
Fine.⚽️

**Review by LiteSun — APPROVED** [review]

**Pull request merged**